.github/workflows/iot-gate-imx8plus-d1d8.yml

```diff
@@ -30,6 +30,11 @@ on:
         required: false
         type: string
         default: balena-staging.com
+      signing-environment:
+        description: The signing environment to use for the bitbake build - includes the related vars and secrets
+        required: false
+        type: string
+        default: ''
```

Comment:
You'll need to provide this input to the downstream workflow, like we do for `deploy-environment`.